appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/jmx/MBeanMetadataHelper.java

```diff
@@ -93,8 +94,13 @@
 
         List<MBeanMetadata> unresolvedMetadataList = resolveDynamicMetadata(metadataList);
         for (MBeanMetadata beanMetadata : metadataList) {
+            List<Tag> tags = new ArrayList<>();
+            for (fish.payara.microprofile.metrics.jmx.XmlTag tag : beanMetadata.getTags()) {
```

Comment:
I'm aware it's moved from slightly further down, but is there a reason this class isn't imported? I don't see a conflict